src/css/hero.css

```diff
@@ -76,14 +81,17 @@
 
 .hero-hashtags {
   display: flex;
+  flex-wrap: wrap;
```

Comment:
<img width="510" alt="image" src="https://github.com/olexandrd/goit-advancedjs-team-3/assets/20472639/e9ecd048-3442-4ad6-bfab-4dfd23f07c18">
Seems like hashtags should be aligned with the text. See the picture.

Reply:
done